H2MLauncher.Core/Services/IGameCommunicationService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Diagnostics;
+
+namespace H2MLauncher.Core.Services;
+
+public interface IGameCommunicationService
```

Comment:
inherit IDisposable if you pretty much explicitly add `Dispose();`

Reply:
Oops auto generated interface :smile: